lib/webauth.js

```diff
@@ -157,7 +163,40 @@ module.exports = function(authManager) {
         result.addHandlerResult(status, handler);
       }
       res.status(200).json(result);
-    },
+    }),
+
+    refreshStatus: Promise.coroutine(function*(req, res) {
+      try {
+        const result = new LoginResult();
+        const handlers = getRelevantHandlers(authManager, req.body);
+        const authServiceHandleMaps = 
+          req[`${UNP.APP_NAME}Data`].webApp.authServiceHandleMaps;
+        for (const handler of handlers) {
+          const pluginID = handler.pluginID;
+          const authPluginSession = getAuthPluginSession(req, pluginID, {});
+          req[`${UNP.APP_NAME}Data`].plugin.services = 
+            authServiceHandleMaps[pluginID];
+          const wasAuthenticate = authPluginSession.authenticated;
```

Comment:
You copied a bug without the FIXME comment

Reply:
OK, will fix